src/Data/Language.hs

```diff
@@ -77,68 +79,61 @@ instance SLanguage 'PHP where
 
 instance FromJSON Language where
   parseJSON = withText "Language" $ \l ->
-    pure $ fromMaybe Unknown (parseLanguage l)
-
-parseLanguage :: Text -> Maybe Language
-parseLanguage l = case T.toLower l of
-  "go"         -> Just Go
-  "haskell"    -> Just Haskell
-  "java"       -> Just Java
-  "javascript" -> Just JavaScript
-  "json"       -> Just JSON
-  "jsx"        -> Just JSX
-  "markdown"   -> Just Markdown
-  "python"     -> Just Python
-  "ruby"       -> Just Ruby
-  "typescript" -> Just TypeScript
-  "php"        -> Just PHP
-  _            -> Nothing
+    pure $ textToLanguage l
```

Comment:
I made our json parsing a little stricter here, languages are case sensitive and must be exactly how they are specified in linguist. Would like some feedback here on if that's OK.